docs/process/prd.md

```diff
@@ -0,0 +1,222 @@
+
+By providing a high quality API backed by a powerful data management solution, ICU4X becomes attractive for segmentation, collation, and unicode regular expression targets which we already see interest from.
+
+Unicode Properties API is also critical for our ability to harness the dynamic Rust community as the GUI/layout needs arise.
+
+### Segmentation
+
+Segmentation aligns with business needs, allowing us to attract Mozilla investment and in return getting ability to test our model in production supplying the needs of both Gecko layout engine and ECMA-402 component.
+
+The segmentation API carries also similar strategic value for the project as the Unicode Set as an attractor for the Rust community, but with a higher-level API allows us to validate performance claims in a performance-critical part of all layout needs which also serves our goal of attracting corporate interest in the project.
+
+### DateTimeFormat
+
+Date and Time formatting are the most common high-level requests out of internationalization APIs and a core part of ECMA-402.
+
+On the technical side, this API allows us to validate our value proposition with a high-level, complex, highly demanded API. We have a chance to explore our modularization capabilities, high-volume data management, and non-trivial algorithmic requirements.
+
+On the business side, we get a chance to supply the most common user needs, and cater to the Internationalization and Rust communities.
+
+### NumberFormat
+
+Number formatting represents a blend of low-level foundational APIs such as Locale, Plural Rules and Unicode Set, with high-level APIs such as DateTimeFormat.
+
+It is required for DateTimeFormat, and will exercise our ability to maintain modularity and lean profile of our APIs, while building a large feature set that [ECMA-402 NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) carries.
+
+### FFI/WASM
+
+FFI poses a lot of technical challenges that we have to overcome to validate the business value of ICU4X being a solution to the high-maintenance cost of keeping separate implementations for many target environments.
+
+We’ll need to validate the ability to produce bindings to other programming languages and technically prove that the DataProvider model can fit those requirements.
+
+## Stretch Goals
+
+With the above listed components, ICU4X 1.0 will provide a comprehensive internationalization solution and validate its value proposition.
+
+Components listed below are worth considering for 1.0 as resources and timing permits, but will not be necessary to prove the product.
+
+### Collator
+
+Collation is one of the core [ECMA-402 components](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator), and builds on top of the Unicode Properties API which we’re investing in.
```

Comment:
question (non-blocking): Doesn't collation also depend on normalization? If so, it would be worth mentioning, and that would also be another reason why collation isn't a quick/easy thing.

Reply:
@sffc ^ do you know? I have never worked on collation impl.

Reply:
I don't feel authoritative answering this question since I have not worked extensively with collation.